src/main/webapp/app/entities/build-log.model.ts

```diff
@@ -57,7 +57,9 @@ export class BuildLogEntryArray extends Array<BuildLogEntry> {
                 // Parse build logs
                 .map(({ log, time }) => ({ log: log.match(this.errorLogRegex), time }))
                 // Remove entries that could not be parsed, are too short or not errors
-                .filter(({ log }: { log: ParsedLogEntry | null; time: string }) => log && log.length === 6 && log[1] === 'ERROR')
+                .filter(({ log }: { log: ParsedLogEntry | null; time: string }) => {
```

Comment:
```suggestion
                .filter(({ log }: { log: ParsedLogEntry | undefined; time: string }) => {
```

Reply:
This will not work I think, see `String.match()` : 
<img width="1107" alt="image" src="https://user-images.githubusercontent.com/41108487/106655217-5c6d4a00-6599-11eb-857f-ae47227989a0.png">

Reply:
ok